auth/user_mgt.go

```diff
@@ -38,6 +39,16 @@ var commonValidators = map[string]func(interface{}) error{
 	"localId":     validateUID,
 }
 
+// Create a new interface
+type headerable interface {
```

Comment:
Thanks for this change!
and I apologize for this secondary nit pick, but I don't like the example names I suggested.
execute would have been a good name if we rolled the ".Do()" line into it. 
as it is maybe "prepareCall", or "setHeader", or "beforeCall" or a similar name that describes the fact that this is a needed step before the call, that adds some params to the header... 

Likewise maybe instead of headerable "relayingpartyCall" would be a better descriptor. 
(again my apologies for not thinking of these names when I suggested the change.)

Reply:
OK, my first idea was setHeader so i keep it.